website/src/util/getSequencesByLocationMapData.ts

```diff
@@ -0,0 +1,93 @@
+import type { MapSource } from '@genspectrum/dashboard-components/util';
+import type { Topology } from 'topojson-specification';
+
+let knownMapFiles: ReturnType<typeof loadKnownMapFiles> | null = null;
+
+/* eslint-disable @typescript-eslint/naming-convention -- the keys must match the country names */
+const mapMetadata: Record<string, Omit<MapData, 'mapSource'>> = {
+    Germany: {
```

Comment:
Is there a way to set the color of the boundaries to black? I just looked at covid sampling in Germany and it is hard to see the borders.
![grafik](https://github.com/user-attachments/assets/a806bd75-0cd1-4683-90ba-89e8d97dfe4a)

Reply:
Hm, true. Or should we use a lighter grey for locations without data?

Reply:
I would be fine with both. I have not tested it. It should just work with all our colors.

Reply:
I'll change it in the components.